src/routes/docs/quick-starts/flutter/+page.markdoc

```diff
@@ -182,35 +182,32 @@ import 'package:appwrite/models.dart' as models;
 
 void main() {
   WidgetsFlutterBinding.ensureInitialized();
-  Client client = Client();
-  client = Client()
+  Client client = Client()
       .setEndpoint("https://cloud.appwrite.io/v1")
-      .setProject("650209298acac4ee1bb6");
+      .setProject("<YOUR_PROJECT_ID>");
   Account account = Account(client);
 
-  runApp(MyApp(
-    account: account,
+  runApp(MaterialApp(
+    home: MyApp(account: account),
   ));
 }
-```
-{% /section %}
-{% section #step-5 step=5 title="Create a login page" %}
-Then, append the following widgets to `lib/main.dart` create your login page.
-import 'package:appwrite/appwrite.dart';
-
-```dart
-class MyForm extends StatefulWidget {
+class MyApp extends StatefulWidget {
   final Account account;
 
-  MyForm({required this.account});
+  MyApp({required this.account});
```

Comment:
If we are renaming this to MyApp, its better to rename `MyFormState` to `MyAppState` as well